script/release.sh

```diff
@@ -229,6 +236,21 @@ for file in $(ls bin/docker-machine-*); do
   fi
 done
 
+display "Uploading checksums"
+for file in sha256sum.txt md5sum.txt; do
```

Comment:
Nitpick, let's please call it `sha256sums.txt` to indicate it contains multiple checksums